core/templates/dev/head/pages/library/Library.js

```diff
@@ -57,6 +57,12 @@ oppia.controller('Library', [
     // Keeps track of the index of the left-most visible card of each group.
     $scope.leftmostCardIndices = [];
 
+    if(['group', 'search'].indexOf($scope.pageMode) !== -1){
```

Comment:
Can you please use LIBRARY_PAGE_MODES constant to maintain consistency (as it's used everywhere in Library.js)!

Reply:
yeah that would be better :)